frontend/src/components/Map/MapDetailBoard.tsx

```
@@ -0,0 +1,70 @@
+import BaseWrapper from '@/components/common/BaseWrapper';
+import Box from '@/components/common/Box';
+import DashBoardHeader from '@/components/common/DashBoardHeader';
+import { Map } from '@/types';
```

Biome

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)